elyra/metadata/schemas/kfp.json

```diff
@@ -53,6 +53,16 @@
             "placeholder": "https://your-kubeflow-service:port/pipeline"
           }
         },
+        "public_api_endpoint": {
+          "title": "Public Kubeflow Pipelines API Endpoint",
+          "description": "Kubeflow Pipelines API Endpoint",
```

Comment:
Nit: the descriptions for the other properties use `The ` to introduce the description, this does not.  We should also lowercase `endpoint` and probably add `public`:
```suggestion
          "description": "The public Kubeflow Pipelines API endpoint",
```